From d98c9717f5b7c7d8ecbed20c9d43db335821c930 Mon Sep 17 00:00:00 2001
From: Matthew Feickert <matthew.feickert@cern.ch>
Date: Wed, 27 Nov 2024 08:57:14 -0700
Subject: [PATCH] docs: Update version number to v1.2.0

* Update the version number in the project metadata to match the release.
   - c.f. https://github.com/peraro/ninja/releases/tag/v1.2.0
---
 README.md    | 2 +-
 VERSION      | 2 +-
 configure.ac | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/README.md b/README.md
index adc1c62..59ff0bf 100644
--- a/README.md
+++ b/README.md
@@ -1,7 +1,7 @@
 Ninja
 =====
 
-- Ninja - version 1.1.0
+- Ninja - version 1.2.0
 - Author: Tiziano Peraro
 
 *Ninja* implements the *Laurent series expansion* method for the
diff --git a/VERSION b/VERSION
index 1cc5f65..867e524 100644
--- a/VERSION
+++ b/VERSION
@@ -1 +1 @@
-1.1.0
\ No newline at end of file
+1.2.0
\ No newline at end of file
diff --git a/configure.ac b/configure.ac
index c9f73fb..d218bc4 100644
--- a/configure.ac
+++ b/configure.ac
@@ -2,7 +2,7 @@
 # Process this file with autoconf to produce a configure script.
 
 AC_PREREQ([2.63])
-AC_INIT([Ninja], [1.1.0], [tiziano.peraro@ed.ac.uk])
+AC_INIT([Ninja], [1.2.0], [tiziano.peraro@ed.ac.uk])
 AC_CONFIG_SRCDIR([src/ninja_implem.cxx])
 AC_CONFIG_HEADERS([config.h src/ninja/ninja_config.h])
 AC_CONFIG_MACRO_DIR([m4])
-- 
2.47.1

